### DISCUSSION TITLE
iOS

### DISCUSSION THREAD

**Discussion by wrwrabbit** [wrwrabbit/Partisan-Telegram-Android#48] [Ideas]
Here's a project we should look into - https://github.com/Postuf/Telegram-iOS-Double-Bottom-Postufgram

**Comment by vivabelarus** [comment]
Было бы хорошо, если бы кто-то изучил их проект. 

Во-первых, они позиционировали своё приложение как демонстрацию концепции двойного дна. А на сколько оно применимо в "боевых" условиях? Меняли ли они логотип на свой? Светятся ли где-то тем, что приложение не является оригиналом?

Во-вторых, есть ли у них ещё что-то кроме двойного дна. Например, у них один из последних коммитов называется "Implement logic to remove chats from hidden account in widget". Правда, удаления чатов я там не увидел. Есть ли у них ещё какой-то функционал, который мог бы быть полезен для нас?

**Reply by wrwrabbit** [reply]
Они просто скрывают, вроде не удаляют

**Reply by vivabelarus** [reply]
То есть никаких фишек кроме скрытия нет? Окей, понял
